src/dds/rs-dds-sensor-proxy.cpp

```diff
@@ -274,7 +274,7 @@ void dds_sensor_proxy::handle_video_data( realdds::topics::image_msg && dds_fram
 {
     frame_additional_data data;  // with NO metadata by default!
     data.timestamp               // in ms
-        = static_cast< rs2_time_t >( realdds::time_to_double( dds_frame.timestamp ) * 1e3 );
+        = static_cast< rs2_time_t >( realdds::time_to_double( dds_frame.timestamp ) * 1e6 );
```

Comment:
I think we test the timestamp in the unit test. How did it pass so far?
https://github.com/IntelRealSense/librealsense/blob/development/unit-tests/dds/test-metadata.py#L194